### PR TITLE
Lock grafana

### PR DESCRIPTION
**Pull request by timja**

### CODE REVIEW

File: helmfile.d/grafana.yaml [hunk 1]
```diff
@@ -9,3 +9,5 @@ releases:
         - "../config/default/grafana.yaml"
       secrets:
         - "../secrets/config/grafana/secrets.yaml"
+bases:
+  - 00-repositories.yaml
```
Comment (halkeye) [inline helmfile.d/grafana.yaml]: What's that file?

Comment (timja) [reply helmfile.d/grafana.yaml]: the repos, apparently the default parent implied way is deprecated, without it, it wouldn't generate the lock file, (i manually edited the file so downgrade, hopefully it works)

**Comment by halkeye** [comment]
https://github.com/dependabot/feedback/issues/174 :( one day one day

**Comment by timja** [comment]
renovate has it (an alternative to dependabot)

**Pull request merged**